v2/tools/generator/internal/config/configuration.go

```diff
@@ -53,6 +53,8 @@ type Configuration struct {
 	// SamplesURL is the URL the samples are accessible at. Paths will be appended to the end of this to
 	// build full sample links. If this is not specified, no samples links are generated.
 	SamplesURL string `yaml:"samplesUrl"`
+	// EmitDocFiles is used as a signal to create doc.go files for packages
```

Comment:
minor: Consider saying "the default is false" (even though it's sorta obvious because this is a non-ptr boolean).